File: Ansh/Array/python/largestNumber.py
class Solution:
    def largestElement(self, nums):
        max=nums[0]
        for i in nums:
            if i>max:
                max=i

        return max
if __name__=="__main__":
    s=Solution()
    print(s.largestElement([1,2,3,4,5,6,7,8,11,9,10,8]))

File: Ansh/Array/java/largestNumber.java
class largestNumber {
    public int largestElement(int[] nums) {
        int max=nums[0];
        for(int i:nums){
            if (i>max){
                max=i;
            }
        }
        return max;
    
    }
    public static void main(String[]args){
        largestNumber s=new largestNumber();
        int[]arr={1,4,2,5,3,6,9,11,7,8};
        int result=s.largestElement(arr);
        System.out.println(result);
    }
}
